cfme/fixtures/automate.py

```diff
@@ -63,3 +67,28 @@ def method(ruby_code):
         return instance
 
     return method
+
+
+@pytest.fixture()
+def import_datastore(appliance, import_data):
+    """This fixture will help to import datastore file
+    Need import_data in tuple like (file_name, from_domain, to_domain)
```

Comment:
Why not define a namedTuple so that tests can consistently and clearly parameterize these, if they aren't named individually.